### PR TITLE
Issue#11

### LINKED ISSUE #11: Chore: Move Content Script Register Into Manifest
Move `registerScript` logic out of service worker (background.js) and move into `manifest.json`

https://github.com/cdrani/yqm/blob/b81f870a08b105ab2d6a4f6c137aa50fcaa2dcdc/background.js#L24-L31

Reference how it's done here: https://github.com/cdrani/chorus/blob/9f07fd960e1742c18cc7dec1054466203543f99d/manifest.json#L16-L23
```json
"content_scripts": [
    {
        "run_at": "document_idle",
        "js": ["utils/state.js", "content-script.js"],
        "css": ["styles.css"],
        "matches": ["*://open.spotify.com/*"]
    }
 ],
```

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fix #11

Hi [cdrani](https://github.com/cdrani),
just trying to help on this issue, please check it out

[Add registerScript on manifest.json](https://github.com/cdrani/yqm/commit/dfa78e75f6ac3502050535ac2020815cf1862df9)

[Delete registerScript from background.js](https://github.com/cdrani/yqm/commit/2e1756c2b260a21962f84751f320ad6b7d4d89eb)

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -21,16 +21,6 @@ async function sendMessage({ message }) {
...
-        id: 'script-content',
-        js: ['content-script.js'],
-        runAt: 'document_idle',
-        matches: ['*://*.youtube.com/*']
-    }])
-}
-
-
 chrome.runtime.onInstalled.addListener(async () => {
     await registerScripts()
```
Comment (cdrani) [inline background.js:25]: This would need to be removed as the function would no longer exist.

File: manifest.json [hunk 2]
```diff
@@ -15,6 +15,11 @@
         "scripting",
         "storage"
     ],
+    "registerScripts": [{
```
Comment (cdrani) [inline manifest.json:18]: The key should be `"content_scripts"`.

File: manifest.json [hunk 3]
```diff
@@ -15,6 +15,11 @@
...
+        "run_at": "document_idle",
+        "js": ['content-script.js'],
+        "matches": ['*://*.youtube.com/*']
```
Comment (cdrani) [inline manifest.json:21]: Not a requirement in json files, but the standard I believe is to just use double quotes **_everywhere_** which would make it easier to comply with the spec so it's something you don't ever need to worry about.

**Comment by cdrani** [comment]
@viktoriussuwandi: Thanks for the PR. I reviewed it and there's some small changes required before I can merge it. However, someone already sent a PR before you and therefore I must give them first priority. As this issue is not really involved, it's most likely they will complete the PR through to merge. In the case they are not able to see it to completion, then you are next in line. Truly sorry and thanks again. 🙏

**Comment by viktoriussuwandi** [comment]
No problem @cdrani :)

**Comment by cdrani** [comment]
> No problem @cdrani :)

Thanks for understanding. If you want to work on something I can assign this issue to you. https://github.com/cdrani/chorus/issues/41

---
Closing this PR as there's another one targeting the same issue. 😭

**Pull request closed without merging**